src/app/components/Modal/FeedbackModal.tsx

```diff
@@ -0,0 +1,187 @@
+"use client";
+
+import { useState } from "react";
+
+type ModalProps = {
+	isVisible: boolean;
+	onClose: () => void;
+	pageTitle: string;
+};
+
+const Modal = ({ isVisible, onClose, pageTitle }: ModalProps) => {
+	const [isQuestionOneDropdown, setIsQuestionOneDropDown] = useState(false);
+	const [isQuestionTwoDropdown, setIsQuestionTwoDropDown] = useState(false);
+
+	const toggleQuestionOneDropdown = () => {
+		setIsQuestionOneDropDown(!isQuestionOneDropdown);
+		if (isQuestionTwoDropdown) {
+			setIsQuestionTwoDropDown(false);
+		}
+	};
+	const toggleQuestionTwoDropdown = () => {
+		setIsQuestionTwoDropDown(!isQuestionTwoDropdown);
+		if (isQuestionOneDropdown) {
+			setIsQuestionOneDropDown(false);
+		}
+	};
+
+	if (!isVisible) return null;
+
+	return (
+		<div className="fixed inset-0 bg-black bg-opacity-50 backdrop-blur-sm flex justify-center items-center">
+			<div className="w-[55%] h-[61%] flex flex-col p-8 rounded-lg shadow bg-[#f9f7fc]">
+				<h1 className="text-2xl font-bold text-gray-800 mb-8">Your Feedback</h1>
+
+				<div className="font-normal text-xs">
+					{/* Display page title*/}
+					<h2 className="mb-2 text-[#805DBE] font-bold">
+						<span className="font-md text-gray-800">
+							You are providing feedback for{" "}
+						</span>
+						{pageTitle}
+					</h2>
+
+					{/* Question One*/}
+					<section>
+						<h2 className="font-medium mt-6 mb-2 text-gray-800">
+							Is this page easy to use?
+						</h2>
+						<div
+							onClick={toggleQuestionOneDropdown}
```

Comment:
Use a button for on click please

Reply:
resolved, FYI: Feedback Modal UI is not 100 percent, most likely will change later on.